src/main/java/com/laa/model/enums/AccountType.java

```diff
@@ -2,5 +2,5 @@
 
 public enum AccountType {
 
-	SINGLE,JOINT
+	SINGLE, JOINT, SAVINGS
```

Comment:
Savings doesnt belong here, perhaps we should name this AccountOwnershipType and put savings in a seperate enum Ownershiptype